cli/upload/upload.go

```diff
@@ -39,35 +40,36 @@ var (
 	importFile string
 	programmer string
 	dryRun     bool
+	tr         = i18n.Tr
 )
 
 // NewCommand created a new `upload` command
 func NewCommand() *cobra.Command {
 	uploadCommand := &cobra.Command{
 		Use:     "upload",
-		Short:   "Upload Arduino sketches.",
-		Long:    "Upload Arduino sketches. This does NOT compile the sketch prior to upload.",
+		Short:   tr("Upload Arduino sketches."),
+		Long:    tr("Upload Arduino sketches. This does NOT compile the sketch prior to upload."),
 		Example: "  " + os.Args[0] + " upload /home/user/Arduino/MySketch",
 		Args:    cobra.MaximumNArgs(1),
 		PreRun:  checkFlagsConflicts,
 		Run:     run,
 	}
 
-	uploadCommand.Flags().StringVarP(&fqbn, "fqbn", "b", "", "Fully Qualified Board Name, e.g.: arduino:avr:uno")
-	uploadCommand.Flags().StringVarP(&port, "port", "p", "", "Upload port, e.g.: COM10 or /dev/ttyACM0")
-	uploadCommand.Flags().StringVarP(&importDir, "input-dir", "", "", "Directory containing binaries to upload.")
-	uploadCommand.Flags().StringVarP(&importFile, "input-file", "i", "", "Binary file to upload.")
-	uploadCommand.Flags().BoolVarP(&verify, "verify", "t", false, "Verify uploaded binary after the upload.")
-	uploadCommand.Flags().BoolVarP(&verbose, "verbose", "v", false, "Optional, turns on verbose mode.")
-	uploadCommand.Flags().StringVarP(&programmer, "programmer", "P", "", "Optional, use the specified programmer to upload.")
-	uploadCommand.Flags().BoolVar(&dryRun, "dry-run", false, "Do not perform the actual upload, just log out actions")
+	uploadCommand.Flags().StringVarP(&fqbn, "fqbn", "b", "", tr("Fully Qualified Board Name, e.g.: arduino:avr:uno"))
+	uploadCommand.Flags().StringVarP(&port, "port", "p", "", tr("Upload port, e.g.: COM10 or /dev/ttyACM0"))
+	uploadCommand.Flags().StringVarP(&importDir, "input-dir", "", "", tr("Directory containing binaries to upload."))
+	uploadCommand.Flags().StringVarP(&importFile, "input-file", "i", "", tr("Binary file to upload."))
+	uploadCommand.Flags().BoolVarP(&verify, "verify", "t", false, tr("Verify uploaded binary after the upload."))
+	uploadCommand.Flags().BoolVarP(&verbose, "verbose", "v", false, tr("Optional, turns on verbose mode."))
+	uploadCommand.Flags().StringVarP(&programmer, "programmer", "P", "", tr("Optional, use the specified programmer to upload."))
+	uploadCommand.Flags().BoolVar(&dryRun, "dry-run", false, tr("Do not perform the actual upload, just log out actions"))
 	uploadCommand.Flags().MarkHidden("dry-run")
 	return uploadCommand
 }
 
 func checkFlagsConflicts(command *cobra.Command, args []string) {
 	if importFile != "" && importDir != "" {
-		feedback.Errorf("error: --input-file and --input-dir flags cannot be used together")
+		feedback.Errorf(tr("error: --input-file and --input-dir flags cannot be used together"))
```

Comment:
Wouldn't a translator risk translating the flags too if we don't do this?
```suggestion
		feedback.Errorf(fmt.Sprintf(tr("error: %s and %s flags cannot be used together"), "--input-file", "--input-dir"))
```

Reply:
I think it's a good change. Thanks.

Taken too far, this sort of thing can make translation more difficult by removing so much of the context from the text that the translator will not be able to understand the intent when reading it as a standalone string on Transifex. But in this particular case, there is no need to know which specific flags the text is referring to.